content/faq/why-should-i-use-blubracket-discover-secrets/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "An example question"
```

Comment:
This and many pages are stubbed out based on the list of FAQs elsewhere. The title and content need to be set.